CMakeLists.txt

```diff
@@ -656,6 +668,13 @@ if(WITH_EXAMPLES)
     set_target_properties(client_examples_${e} PROPERTIES RUNTIME_OUTPUT_DIRECTORY ${CMAKE_CURRENT_BINARY_DIR}/examples/client)
     target_link_libraries(client_examples_${e} vncclient ${CMAKE_THREAD_LIBS_INIT} ${SDL2_LIBRARY} ${GTK2_LIBRARIES} ${FFMPEG_LIBRARIES} ${LIBSSHTUNNEL_LIBRARY})
   endforeach(e ${LIBVNCCLIENT_EXAMPLES})
+
+  if(Qt5Widgets_FOUND AND WITH_QT)
```

Comment:
What about adding qt5client to LIBVNCCLIENT_EXAMPLES in line with the other examples and use Qt5Widgets_LIBRARIES ?

Reply:
I tried to add this example to LIBVNCCLIENT_EXAMPLES, but CMake would not find it because the extension is .cpp, and not .c. If you rename the source file to qt5client.c, it will not link to the C++ libraries. The simpler solution that I found was to write these "special" build instructions to CMakeLists.
Are you suggesting to replace "Qt5::Widgets" in the target_link_libraries macro with "Qt5Widgets_LIBRARIES"? It seems like that's an alias to "Qt5::Widgets"... Can you explain your suggestion?

Reply:
OK got it. Can you add a comment above that it's about C vs C++ please?

Re the Qt5::Widgets" I meant using that or an equivalent up in lines 653ff (the LIBVNCCLIENT_EXAMPLES handling) but as this is not feasible, you can leave as-is.

Reply:
Done, check [d7ae226](https://github.com/LibVNC/libvncserver/pull/612/commits/d7ae2265f8c68ca5b43d238d56af9dda1d1e2062). 

So, it is totally feasible to change Qt::Widgets to Qt5Widgets_LIBRARIES, I did it in [9a20885](https://github.com/LibVNC/libvncserver/pull/612/commits/9a208856142223fcf0f96c5a708fbae7658fa84c) to follow the "*_LIBRARIES" standard, as you did with the other libraries.